cfgov/wellbeing/jinja2/wellbeing/home.html

```diff
@@ -20,6 +20,8 @@
 
 {% block css -%}
     {{ super() }}
+    <link rel="stylesheet"
+    href="{{ static('apps/financial-well-being/css/main.css') }}">
```

Comment:
TRIVIAL: We usually try to indent (ideally align the attributes—so aligning `rel` and `href`) of the second line of a tag that spans multiple lines. See the `<input …>`s in this file, for example. Some day we'll implement a linter for jinja and have that take care of it.

Reply:
Ok. Not sure if I had it that way originally, but I needed to disable prettier to get it to line up.